index.html

```diff
@@ -5,16 +5,83 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/grid.css"> 
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
-
 <body>
+<div class="container">
+    <div class="row"> <!--header-->
+        <header>
+            <div class="col-6">
+            <h1 class="logo-title">ALLAN JOHNSON</h1>
+            <h3 class="logo-sub">Personal Life Coach</h3>
+        </div>
+            <nav>
+                <div class="col-6">
+                <ul>
+                    <li>About</li>
+                    <li>Services</li>
+                    <li>Motivational Reading</li>
+                    <li>Blog</li>
+                    <li>Contact</li>
+                    <li>Log In</li>
+               </ul>
+        </nav>
+        </header>
+    </div>
 
-    <!-- Add your markup here -->
+    <div class="row"> <!--hero-->
+        <div class="col-12 hero">
+            <p class="ambition">Ambition is the first step towards</p>
+            <p class="success">SUCCESS</p>
+            <button class="book-now" type="button">Book Now</button>
+        </div>
+    </div>
+    <div class="row"> <!--About Me-->
+       <div class="row3">
```

Comment:
Less good here. 2 things:
1) As mentioned above, don't add any elements between the `row` and the `col-6`. `col-6` should be a direct child of `row`.
2) Avoid re-using class names that are used in the grid system to avoid confusion. If they're your own classes, give them their own name 🙂

Reply:
If you do this (and by doing that, get rid of `row3`, it will fix the issue you have with both left and right sections stacking vertically.